aunt-leahs-app/src/pages/VolunteerLoginPage.jsx

```diff
@@ -14,26 +14,29 @@ import { setCurrentPage } from '../redux/page/pageAction';
 import { useEffect } from 'react';
 
 
-function VolunteerLoginPage({ history, setExistingVolunteer, setCurrentPage }) {
+function VolunteerLoginPage({ setExistingVolunteer, setCurrentPage }) {
 	const [volunteers, setVolunteers] = useState([]);
 
 	useEffect(() => {
 		getVolunteers();
 	}, []);
 
 	async function getVolunteers() {
-		const response = await fetch('http://localhost:7071/api/VolunteerNames');
-		const volunteers = await response.json();
-		const items = [];
+		try {
+			const response = await fetch('http://localhost:7071/api/VolunteerNames');
 
-		volunteers.forEach(volunteer => {
-			const volunteerObj = {
-				id: volunteer.volunteer_id,
-				value: volunteer.first_name + " " + volunteer.last_name
-			}
-			items.push(volunteerObj);
-		})
-		setVolunteers(items);
+			const volunteers = await response.json();
+			const volunteerNames = volunteers.map(volunteer => {
+				return {
+					id: volunteer.volunteer_id,
+					value: volunteer.first_name + " " + volunteer.last_name
+				}
+			})
```

Comment:
You can just put the object in brackets to return it without the extra braces and entire return statement. It makes it a little cleaner imo.

See: https://stackoverflow.com/questions/28770415/ecmascript-6-arrow-function-that-returns-an-object